string name
float32 activation
Correlation[] correlations
float32 satisfaction
float32 threshold
Wish[] wishes
bool isExecuting
int32 executionTimeout
int32 executionTime
bool active
bool interruptable
bool independentFromPlanner
int32 priority
bool activated
bool executable